src/NuGet.Client/project.json

```diff
@@ -1,15 +1,15 @@
 ﻿{
-    "version": "3.1.2-*",
+    "version": "3.2-*",
     "compilationOptions": {
         "warningsAsErrors": true
     },
     "dependencies": {
-        "NuGet.Versioning": "3.1.2-*",
-        "NuGet.Packaging": "3.1.2-*",
-        "NuGet.Repositories": "3.1.2-*",
-        "NuGet.RuntimeModel": "3.1.2-*",
-        "NuGet.ContentModel": "3.1.2-*",
-        "NuGet.Common": { "version": "3.1.2-*", "type": "build" }
+        "NuGet.Versioning": "3.2-*",
```

Comment:
@davidfowl changing versions is painful. in msbuild we can use a macro for these values. Is there a mechanism like that in project.json ?